app/views/case_workers/claims/_claims_list.html.haml

```diff
@@ -29,8 +29,9 @@
       - present_collection(claims).each do |claim|
         %tr
           %td
-            = link_to claim.case_number , case_workers_claim_path(claim),{:class => 'js-test-case-number-link'}
-            %div.unique-id-small= claim.unique_id
+            = link_to claim.case_number , case_workers_claim_path(claim),{:class => 'js-test-case-number-link', 'aria-label': "#{claim.state.humanize}, claim I.D. #{claim.unique_id}"}
```

Comment:
Hey @cleargif, wouldn't be better to use the case number instead of the database ID in the are-label?
The database ID (unique_id) is not going to be very meaningful to a screen reader user.